docs/guide/custom-functions.md

```diff
@@ -262,6 +262,19 @@ MyFunctionPlugin.aliases = {
 };
 ```
 
+::: tip
+Aliases also require translations. Otherwise, they will not work.
```

Comment:
Better:
For each alias of your function, define a translation, even if you want to support only one language.

Thanks to @kirszenbaum

Reply:
Ok. I went with @kirszenbaum's option